virt-v2v/cold/entrypoint.go

```diff
@@ -127,6 +128,19 @@ func buildCommand() []string {
 				virtV2vArgs = append(virtV2vArgs, "--mac", macToIp)
 			}
 		}
+		// Adds LUKS keys, if exist.
+		if file, err := os.Open("/etc/luks/luks"); err == nil {
```

Comment:
maybe I'm misreading, but it looks like we would ignore the error, no? I suppose we want to log it at least if it's not related to the file being missing